editor/components/writing-flow/index.js

```diff
@@ -160,8 +179,27 @@ class WritingFlow extends Component {
 		return editables.length > 0 && index === edgeIndex;
 	}
 
+	/**
+	 * Function called to ensure the block parent of the target node is selected.
+	 *
+	 * @param {DOMElement} target
+	 */
+	selectParentBlock( target ) {
+		if ( ! target ) {
+			return;
+		}
+
+		const parentBlock = target.hasAttribute( 'data-block' ) ? target : target.closest( '[data-block]' );
```

Comment:
FYI: `Element#closest` includes the node itself in the test, so the ternary is redundant.

>The `Element.closest()` method returns the closest ancestor of the current element **(or the current element itself)** which matches the selectors given in parameter. If there isn't such an ancestor, it returns null.

https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

```
document.body.closest( 'body' ) === document.body;
// true
```